learn/tooling-guide/cli-tools/openapi.md

```diff
@@ -23,36 +23,32 @@ The OpenAPI tools provide the following capabilities.
  2. Export the OpenAPI definition of a Ballerina service.
  3. Validate service implementation of a given OpenAPI Contract.
     
-The `openapi` command in Ballerina is used for OpenAPI to Ballerina and Ballerina to OpenAPI code generations. 
-Code generation from OpenAPI to Ballerina can produce `ballerina service stubs` and `ballerina client stubs`.
-The OpenAPI compiler plugin will allow you to validate a service implementation against an OpenAPI contract during
- compile time. 
-This plugin ensures that the implementation of a service does not deviate from its OpenAPI contract.   
+The `openapi` command in Ballerina is used for OpenAPI to Ballerina and Ballerina to OpenAPI code generations. Code generation from OpenAPI to Ballerina can produce `ballerina service stubs` and `ballerina client stubs`.
```

Comment:
"ballerina service stubs" and "ballerina client stubs" aren't code. So shouldn't be formatted as such. And isn't it redundant to add the "ballerina" prefix to them? 🤔 Wouldn't just "... can produce service stubs and client stubs" do? 🤔

Reply:
Fixed.